python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -23,6 +23,7 @@
 import sgtk
 from sgtk.commands.clone_configuration import clone_pipeline_configuration_html
 from sgtk.util import process
+from . import constants
```

Comment:
I personally don't like this pattern of defining constants in a global file. This make code harder to update when cleaning stuff up. Until someone convinces me otherwise, that's one pattern I'd love for us to kill to be honest in Toolkit. I don't care enough about it right now to ask for changes, but to be honest if there's no benefit from moving things around I'd ague for leaving constants as they were.